prac_06/languages.py

```diff
@@ -0,0 +1,16 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+print(python)
+
```

Comment:
If you use a function and call another variable outside the function, that variable is a global constant, so python, ruby and visual_basic should be in capital letters. It is not recommended to use global variables.